drivers/entropy/entropy_stm32.c

```diff
@@ -36,6 +36,14 @@
 
 #if defined(RNG_CR_CONDRST)
 #define STM32_CONDRST_SUPPORT
+
+/* These calls are named differently in the STM32L4 LL RNG driver */
+#if defined(STM32L4)
+#define LL_RNG_EnableCondReset(x) LL_RNG_SetConditioningResetBit(x)
+#define LL_RNG_DisableCondReset(x) LL_RNG_ResetConditioningResetBit(x)
+#define LL_RNG_IsEnabledCondReset(x) LL_RNG_IsResetConditioningBitSet(x)
```

Comment:
Please avoid function redefinitions. I would prefer you use intermediate functions like:
```
void ll_func_enable_cond_reset {
#ifdef !CONFIG_SOC_SERIES_STM32L4X
       LL_RNG_EnableCondReset(x)
#else
       LL_RNG_SetConditioningResetBit(x)
#endif
}
```
I understand the advantage is that current change is much more concise.
Though, i'd like to avoid having redefinitions poping up in each file which would end up to be quite confusing in the end (which one should be redefined ? Is that a one to one redefinition? ... )